src/riscv.c

```diff
@@ -25,6 +25,10 @@ void block_map_clear(block_map_t *map)
         block_t *block = map->map[i];
         if (!block)
             continue;
+        for (uint32_t i = 0; i < block->n_insn; i++) {
+            if (block->ir[i].fuse)
```

Comment:
Simply remove `if (block->ir[i].fuse)` since [free(3p)](https://man7.org/linux/man-pages/man3/free.3p.html) would check.
> If ptr is a null pointer, no action shall occur.